tests/test_core.py

```diff
@@ -422,6 +424,69 @@ def test_response(self, spec):
                 == metadata["components"]["responses"]["test_response"]
             )
 
+
+    def test_response_with_HTTPStatus_code(self, spec):
+        if int(sys.version[0]) == 2:
```

Comment:
You should use [`@pytest.mark.skip`](https://docs.pytest.org/en/latest/skipping.html) for that.

Reply:
Slight tweak - using ```@pytest.mark.skipif``` instead.